src/jaxoplanet/orbits/keplerian.py

```diff
@@ -70,6 +87,19 @@ def from_orbital_properties(
         radius: Optional[Quantity] = None,
         body_mass: Optional[Quantity] = None,
     ) -> "Central":
+        """Initialize the central body (e.g. a star) of an orbital system using orbital parameters to derive radius
+        and mass.
+
+        Args:
+            period: The orbital period of the orbiting body [time unit].
+            semimajor: The semi-major axis of the orbiting body [length unit].
+            radius (Optional[Quantity]): Radius of central body [length unit].
+            body_mass (Optional[Quantity]): Mass of orbiting body [mass unit].
+
+        Returns:
+            Central object
+        """
+        # Check that input semimajor is a scalar - what about period/radius/body mass?
```

Comment:
It seems like scalar checks of other input parameters was missing here

Reply:
I thought there was a reason for this, but I can't remember for the life of me. It looks like adding a check for `period` and `body_mass` would make sense. I don't think we need the check for radius because that happens in the `__init__` above.

Reply:
Please feel free to add that check as part of this PR!

Reply:
Added